3.resistance-signature/scripts/nbconverted/11.1.last-batch-visualize-otherclone-signature-application_validation.r

```diff
@@ -0,0 +1,190 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(cowplot))
+suppressPackageStartupMessages(library(gridExtra))
+
+source(file.path("utils", "viz.R"))
+
+seed <- 1234
+set.seed(seed)
+
+# Set constants
+dataset <- "otherclones"
+sig_dir <- file.path("results", "singscore")
+perf_dir <- file.path("results", "performance")
+
+sig_results_file <- file.path(sig_dir, paste0("singscore_results_LAST_BATCH_VALIDATIONotherclones.tsv.gz"))
+roc_auc_file <- file.path(perf_dir, paste0(dataset, "_bortezomibsignature_roc_auc_LAST_BATCH_VALIDATION.tsv"))
+roc_curve_file <- file.path(perf_dir, paste0(dataset, "_bortezomibsignature_roc_curve_LAST_BATCH_VALIDATION.tsv"))
+
+output_figure_dir <- file.path("figures", "singscore")
+
+# Load signature scores
+# Load raw signature results
+sig_cols <- readr::cols(
+    .default = readr::col_character(),
+    Metadata_clone_type_indicator = readr::col_integer(),
+    Metadata_celltype_shorthand_from_plate_graph = readr::col_integer(),
+    Metadata_date = readr::col_integer(),
+    Metadata_treatment_shorthand_from_plate_graph = readr::col_integer(),
+    TotalScore = readr::col_double(),
+    TotalDispersion = readr::col_double(),
+    UpScore = readr::col_double(),
+    UpDispersion = readr::col_double(),
+    DownScore = readr::col_double(),
+    DownDispersion = readr::col_double(),
+    Metadata_permuted_p_value = readr::col_double(),
+    min_permuted_value = readr::col_double(),
+    max_permuted_value = readr::col_double()
+)
+
+#inference_df <- readr::read_tsv(sig_results_file, col_types = sig_cols) %>%
+    #dplyr::filter(Metadata_model_split == "inference")
+
+signature_df <- readr::read_tsv(sig_results_file, col_types = sig_cols) %>%
```

Comment:
```r
signature_df <- readr::read_tsv(sig_results_file, col_types = sig_cols)
```

Since you're not filtering inference, you need to delete this dplyr pipe

Reply:
remember, you should make this change in the `.ipynb` file.